presto-main/src/test/java/com/facebook/presto/sql/planner/TestDomainTranslator.java

```diff
@@ -1072,6 +1072,24 @@ public void testFromInPredicate()
 //        assertTrue(result.getTupleDomain().isNone());
     }
 
+    @Test
+    public void testFromInPredicateWithCasts()
+    {
+        Expression originalExpression = new InPredicate(
```

Comment:
what about cases where there is `null` value in list (see commented code above), e.g:
```
A in (1L, 2L, null);
not (A in (1L, 2L, null);
A in (null);
not (A in (null));
```